includes/Checker/Check.php

```diff
@@ -15,6 +15,30 @@
  * @since n.e.x.t
  */
 interface Check {
+	/**
+	 * Flag for stable checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const STABILITY_STABLE = 1;
+
+	/**
+	 * Flag for experimental checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const STABILITY_EXPERIMENTAL = 2;
+
+	/**
+	 * Returns the checks stability.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return int
+	 */
+	public function get_stability();
```

Comment:
```suggestion
	/**
	 * Returns the check's stability.
	 *
	 * @since n.e.x.t
	 *
	 * @return string One of the check stability constant values.
	 */
	public function get_stability();
```

Reply:
Thanks @felixarntz this has been updated